benchmarks/infill_density/infill_ascii.prm

```diff
@@ -0,0 +1,199 @@
+# This model demonstrates ASPECT's ability to be used to
+# model load induced flexure from a topographic feature
+# on top of the lithosphere. The model uses the viscoelastic
+# material model and is composed of an 'elastic' high viscosity
+# lithosphere and a low viscosity mantle beneath the elastic
+# lithosphere. A time dependent axisymmetric box load is applied
+# to the top boundary using ASCII files, with the center of the load
+# at the left boundary of the 2D cartesian model domain. The model
+# is run for 120'000 years, which is equivalent to 40*tm:
+# tm = eta_m / mu
+# eta_m = mantle viscosity
+# mu = elastic shear modulus
+# tm is the time scale of viscous stress relaxation for the mantle.
+# This allows for virtually all stresses in the mantle to be relaxed
+# and for the elastic lithosphere to be fully compensating the load.
+# The analytic solution for this setup is outlined in the Generic
+# Mapping Tools function grdflexure:
+# https://docs.generic-mapping-tools.org/6.1/supplements/potential/grdflexure.html
+
+#  Global parameters
+set Additional shared libraries            = ./libinfill_density.so
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 120e3
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.1
+set Maximum time step                      = 750
+set Output directory                       = output
+set Pressure normalization                 = no
+
+# Model geometry (500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 15
+    set Y repetitions = 30
+    set X extent      = 1200e3
+    set Y extent      = 1200e3
+  end
+end
+
+# Mesh refinement specifications
+# Refine the mesh in the uppermost part of the model
+# to better capture lithospheric response to the load.
+subsection Mesh refinement
+  set Initial adaptive refinement              = 2
+  set Initial global refinement                = 1
+  set Time steps between mesh refinement       = 1
+  set Minimum refinement level                 = 0
+  set Strategy                                 = minimum refinement function, strain rate
+  subsection Minimum refinement function
+    set Coordinate system                      = cartesian
+    set Variable names                         = x,y,t
+    set Function expression                    = if(y>=1050e3, 3, 0)
+  end
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree           = 2
+  set Stokes velocity polynomial degree       = 2
+  set Temperature polynomial degree           = 1
+  set Use locally conservative discretization = true
```

Comment:
I remember it increasing the accuracy yea, and my assumptions for why that was the case is what you added as a comment further down about better tracking for the lithosphere (since the analytic solution depends on lithosphere thickness).

Reply:
This parameter is actually independent from the DG composition formulation, but let's check it's affect later (probably has little to no influence).